optaplanner-benchmark/src/test/resources/org/optaplanner/benchmark/api/classloaderTestdataBenchmarkConfig.xml

```diff
@@ -16,7 +16,9 @@
 
       <!-- Score configuration -->
       <scoreDirectorFactory>
-        <scoreDrl>divertThroughClassLoader/org/optaplanner/benchmark/api/classloaderTestdataConstraints.drl</scoreDrl>
+        <!-- Using these classnames doesn't work because the className still differs from class.getName()-->
+        <!--<constraintProviderClass>divertThroughClassLoader.org.optaplanner.core.impl.testdata.domain.TestdataConstraintProvider</constraintProviderClass>-->
+        <constraintProviderClass>org.optaplanner.core.impl.testdata.domain.TestdataConstraintProvider</constraintProviderClass>
```

Comment:
One can argue wheter the PlannerBenchmarkFactoryTest shouldn't check the DRL redirect too, but I'd argue it's more important to at least test the ConstraintProvider.

As for the comment: same limitations as for the entityClass etc a few lines above it.

Reply:
This test is probably too convoluted anyway. It tests if a classloader works correctly, by checking if a redirect in the custom class loader succeeds, without going directly to the correct resource in the parent class...